client/state/data-layer/wpcom/concierge/schedules/appointments/index.js

```diff
@@ -23,6 +27,28 @@ export const toApi = ( { beginTimestamp, customerId, siteId, meta } ) => ( {
 	meta: JSON.stringify( meta ),
 } );
 
+export const cancelConciergeAppointment = ( { dispatch }, action ) => {
+	dispatch(
+		http(
+			{
+				method: 'POST',
+				path: `/concierge/schedules/${ action.scheduleId }/appointments/${ action.appointmentId }/cancel`,
+				apiNamespace: 'wpcom/v2',
+				body: {},
+			},
+			action
+		)
+	);
+};
+
+export const markSlotAsCancelled = ( { dispatch } ) =>
+	dispatch( successNotice( translate( 'Your appointment has been successfully cancelled.' ) ) );
+
+export const handleCancellingError = ( { dispatch } ) =>
+	dispatch(
+		errorNotice( translate( 'We could not cancel your appointment. Please try again later.' ) )
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'We could not book your appointment. Please try again later.' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).